hack/cmd_util.sh

```diff
@@ -174,7 +174,8 @@ function os::cmd::internal::expect_exit_code_run_grep() {
 	os::test::junit::declare_test_start
 
 	local name=$(os::cmd::internal::describe_call "${cmd}" "${cmd_eval_func}" "${grep_args}" "${test_eval_func}")
-	echo "Running ${name}..."
+    local preamble="Running ${name}..."
```

Comment:
@deads2k please direct commentary to https://github.com/openshift/origin/pull/6598